protonj2-client/src/main/java/org/apache/qpid/protonj2/client/TransportOptions.java

```diff
@@ -329,20 +329,20 @@ public TransportOptions allowNativeIO(boolean allowNativeIO) {
     }
 
     /**
-     * @return the nativeIOPeference
+     * @return the nativeIOPreference
      */
-    public String[] nativeIOPeference() {
-        return nativeIOPeference;
+    public String[] nativeIOPreference() {
+        return nativeIOPreference;
     }
 
     /**
-     * @param nativeIOPeference the nativeIOPeference to set
+     * @param nativeIOPreference the nativeIOPreference to set
      */
-    public void nativeIOPeference(String... nativeIOPeference) {
-        if (nativeIOPeference == null || nativeIOPeference.length == 0 || nativeIOPeference.length == 1 && nativeIOPeference[0] == null) {
-            this.nativeIOPeference = DEFAULT_NATIVEIO_PREFERENCES;
+    public void nativeIOPreference(String... nativeIOPreference) {
+        if (nativeIOPreference == null || nativeIOPreference.length == 0 || nativeIOPreference.length == 1 && nativeIOPreference[0] == null) {
+            this.nativeIOPreference = DEFAULT_NATIVEIO_PREFERENCES;
         } else {
-            this.nativeIOPeference = nativeIOPeference;
+            this.nativeIOPreference = nativeIOPreference;
```

Comment:
public api change

Reply:
API changes are ok at this point as we are still in milestone releases and not an official 1.0

Reply:
This was addressed with larger API docs changes in https://gitbox.apache.org/repos/asf?p=qpid-protonj2.git;h=acfb425